tests/testTriggers.go

```diff
@@ -9933,6 +9947,403 @@ func TriggerReallocSharedMount(contexts *[]*scheduler.Context, recordChan *chan
 	})
 }
 
+// TriggerCreateAndRunFioOnVcluster creates and runs fio on vcluster
+func TriggerCreateAndRunFioOnVcluster(contexts *[]*scheduler.Context, recordChan *chan *EventRecord) {
+	defer ginkgo.GinkgoRecover()
+	defer endLongevityTest()
+	startLongevityTest(CreateAndRunFioOnVcluster)
+	event := &EventRecord{
+		Event: Event{
+			ID:   GenerateUUID(),
+			Type: CreateAndRunFioOnVcluster,
+		},
+		Start:   time.Now().Format(time.RFC1123),
+		Outcome: []error{},
+	}
+	defer func() {
+		event.End = time.Now().Format(time.RFC1123)
+		*recordChan <- event
+	}()
+
+	setMetrics(*event)
+
+	vc := &vcluster.VCluster{}
+	var scName string
+	var pvcName string
+	var appNS string
+	fioOptions := vcluster.FIOOptions{
+		Name:      "mytest",
+		IOEngine:  "libaio",
+		RW:        "randwrite",
+		BS:        "4k",
+		NumJobs:   1,
+		Size:      "500m",
+		TimeBased: true,
+		Runtime:   "600s",
```

Comment:
Keep the runtime as configurable - maybe reading from configmap. Keep default as 1 day,

Reply:
Done. defined the variable to take default as 86,400 seconds and have configured a function to read the data if present in config map.